src/Entities/Entity.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+namespace UKFast\SDK\Entities;
+
+abstract class Entity
+{
+    protected $guarded = [];
```

Comment:
It would be nice to have a docblock here, as decided when we came up with the entity filling idea